### DISCUSSION TITLE
How to run the csg_to_cad example inside docker container without skinner error

### DISCUSSION THREAD

**Discussion by Minhajur-NE-25** [neams-th-coe/cardinal#982] [Q&A] [answered]
Hello altruists,
I am trying to run the csg_to_cad example inside the docker container downloaded using the 
[Docker.Instructions.2.pdf](https://github.com/user-attachments/files/17674310/Docker.Instructions.2.pdf)

As the skinner capability to csg model has been added by a recent commit, the docker is unaware of that. So, the question is can I bypass this error without installing cardinal from ground up? I am actually trying to avoid going through the installataion process  :)

*** ERROR ***
/workdir/cardinal/tutorials/csg_to_cad/openmc.i:3.1:
The following error occurred in the Problem 'MOOSE Problem' of type OpenMCCellAverageProblem.

The 'skinner' can only be used with OpenMC geometries that are entirely DAGMC based.
Your model contains a combination of both CSG and DAG cells.

**Comment by aprilnovak** [comment]
Hi @Minhajur-NE-25, great question! We recently added the skinning capability for hybrid OpenMC models, and it appears the Docker container needs to be updated. @loganharbour, what is the process for updating the Docker container to the last Cardinal version?

**Reply by loganharbour** [reply]
I'm working on a release. One should be out this evening.

**Reply by Minhajur-NE-25** [reply]
Hi @loganharbour, if you are done with updating the dockerfile, could you please share the link to the recent release?

**Reply by loganharbour** [reply, marked as answer]
It's available now in `idaholab/cardinal:2024.11.09-33a8f9c` [here](https://hub.docker.com/layers/idaholab/cardinal/2024.11.09-33a8f9c/images/sha256-e19116710210d9299f7d32f9e47fa8303aabc21e61499103642ab325124a8a25?context=repo)

Please feel free to reach out with any issues
